{{cookiecutter.plugin_name}}/tox.ini

```diff
@@ -36,3 +36,6 @@ deps =
     qtpy
     pyqt5
 commands = pytest -v --color=yes --cov={{cookiecutter.module_name}} --cov-report=xml
+
+[flake8]
+max-line-length=88
```

Comment:
I know @jni hates black's line length of 88, so rather than tell flake to make it 88, let's tell black to make it 79 :)

put it in pyproject.toml instead of tox.ini:

```toml
[tool.black]
line-length = 79
```

Reply:
actually, while you're in there, add some isort settings too:

```
[tool.isort]
profile = "black"
line_length = 79
```